lighttpd.conf

```diff
@@ -0,0 +1,46 @@
+mimetype.assign = (
+  ".html" => "text/html",
+  ".htm" => "text/html",
+  ".shtml" => "text/html",
+  ".css" => "text/css",
+  ".gif" => "image/gif",
+  ".jpeg" => "image/jpeg",
+  ".jpg" => "image/jpeg",
+  ".js" => "application/javascript",
+  ".txt" => "text/plain",
+  ".png" => "image/png",
+  ".ico" => "image/x-icon",
+  ".svg" => "image/svg+xml",
+  ".webp" => "image/webp",
+  ".json" => "application/json",
+)
+
+server.username = "www-data"
+server.groupname = "www-data"
+
+server.max-fds = 8192
+server.max-connections = 4096
+server.event-handler = "linux-sysepoll"
+server.network-backend = "sendfile"
+
+server.max-worker = 2
```

Comment:
[documentation](https://redmine.lighttpd.net/projects/lighttpd/wiki/Server_max-workerDetails) states that configuring this to `0` is *strongly* recommended. again: documentation might be outdated. but assuming it is not: is there a reason why you believe we should configure a different value than the recommendation?